src/app/components/LocationCard/LocationCard.tsx

```diff
@@ -1,22 +1,23 @@
 import React from 'react';
+import useRevGeoCoding from '../../hooks/useRevGeoCoding';
 import styles from './LocationCard.module.css';
 
 export type LocationCardProps = {
-  locationName: string;
   degree: number;
   icon: string;
   description: string;
 };
 
 export default function LocationCard({
-  locationName,
   degree,
   icon,
   description,
 }: LocationCardProps): JSX.Element {
+  const cityName = useRevGeoCoding();
+
```

Comment:
No please don't move it into the locationCard but rather to the top of the Page function